generate-stackbrew-library.sh

```diff
@@ -154,21 +163,32 @@ for version; do
 	variants="$(jq -r '.[env.version].variants | map(@sh) | join(" ")' versions.json)"
 	eval "variants=( $variants )"
 
-	for variant in "${variants[@]}"; do
-		dir="$version${variant:+/$variant}"
+	for v in "${variants[@]}"; do
+		dir="$version${v:+/$v}"
 		[ -f "$dir/Dockerfile" ] || continue
 
 		commit="$(dirCommit "$dir")"
 
+		variant="$(basename "$v")"
 		variantAliases=( "${versionAliases[@]/%/${variant:+-$variant}}" )
 		variantAliases=( "${variantAliases[@]//latest-/}" )
 
+		sharedTags=()
```

Comment:
Do we want shared tags between Windows & Linux too? (like python)

Reply:
We didn't previously, which is why I didn't here -- I think that was a conscious choice due to the massive differences in their ability to be functional/useful on the different platforms.